File: src/main/java/top/zbeboy/isy/web/bean/graduate/design/replan/DefenseGroupMemberBean.java
package top.zbeboy.isy.web.bean.graduate.design.replan;

import lombok.Data;
import lombok.EqualsAndHashCode;
import top.zbeboy.isy.domain.tables.pojos.DefenseGroupMember;

/**
 * Created by zbeboy on 2017/7/18.
 */
@EqualsAndHashCode(callSuper = false)
@Data
public class DefenseGroupMemberBean extends DefenseGroupMember {
    private String studentNumber;
    private String studentName;
    private String studentMobile;
    private String subject;
    private String staffName;
    private int studentId;
}

File: src/main/java/top/zbeboy/isy/web/bean/graduate/design/replan/DefenseGroupMemberBean.kt
package top.zbeboy.isy.web.bean.graduate.design.replan

import top.zbeboy.isy.domain.tables.pojos.DefenseGroupMember

/**
 * Created by zbeboy 2018-02-06 .
 **/
class DefenseGroupMemberBean : DefenseGroupMember() {
    var studentNumber: String? = null
    var studentName: String? = null
    var studentMobile: String? = null
    var subject: String? = null
    var staffName: String? = null
    var studentId: Int = 0
}